patches/@mui+material+5.18.0.patch

```diff
@@ -0,0 +1,65 @@
+diff --git a/node_modules/@mui/material/TablePagination/TablePagination.d.ts b/node_modules/@mui/material/TablePagination/TablePagination.d.ts
+index eddd787..b44d025 100644
+--- a/node_modules/@mui/material/TablePagination/TablePagination.d.ts
++++ b/node_modules/@mui/material/TablePagination/TablePagination.d.ts
+@@ -13,6 +13,7 @@ export interface LabelDisplayedRowsArgs {
+   to: number;
+   count: number;
+   page: number;
++  of?: string;
+ }
+ 
+ /**
+@@ -154,6 +155,7 @@ export interface TablePaginationOwnProps extends TablePaginationBaseProps {
+    * The system prop that allows defining system overrides as well as additional CSS styles.
+    */
+   sx?: SxProps<Theme>;
++  of?: string;
+ }
+ 
+ export interface TablePaginationTypeMap<AdditionalProps, RootComponent extends React.ElementType> {
+diff --git a/node_modules/@mui/material/TablePagination/TablePagination.js b/node_modules/@mui/material/TablePagination/TablePagination.js
+index a57a7f5..adecbb4 100644
+--- a/node_modules/@mui/material/TablePagination/TablePagination.js
++++ b/node_modules/@mui/material/TablePagination/TablePagination.js
+@@ -115,9 +115,10 @@ const TablePaginationDisplayedRows = styled('p', {
+ function defaultLabelDisplayedRows({
+   from,
+   to,
+-  count
++  count,
++  of = 'of'
+ }) {
+-  return `${from}–${to} of ${count !== -1 ? count : `more than ${to}`}`;
```

Comment:
we're patching material to make `of` translatable?

Reply:
I removed the previous version of the patch, I think the diff looks better now.